### PR TITLE
Deploy docs under 0.7

### PR DESCRIPTION
**Pull request by mschauer**
0.6 is not run anymore.

### CODE REVIEW

**Comment by mschauer** [comment]
By the way, `.travis.yml` contains a reference to 0.6 which is dead code. Should `Coverage` be run through travis?

**Comment by fredrikekre** [comment]
Maybe enable 1.0 in the build matrix and build docs there?

File: .travis.yml [hunk 1]
```diff
@@ -21,8 +22,8 @@ branches:
...
 after_success:
-  - if [ $TRAVIS_JULIA_VERSION = "0.6" ] && [ $TRAVIS_OS_NAME = "linux" ]; then
-      julia -e 'cd(Pkg.dir("StaticArrays")); Pkg.add("Coverage"); using Coverage; Coveralls.submit(Coveralls.process_folder()); Codecov.submit(Codecov.process_folder())';
+  - if [ $TRAVIS_JULIA_VERSION = "0.7" ] && [ $TRAVIS_OS_NAME = "linux" ]; then
+      julia -e 'using Pkg; Pkg.add("Coverage"); using Coverage; Coveralls.submit(Coveralls.process_folder()); Codecov.submit(Codecov.process_folder())';
     fi
-  - julia -e 'Pkg.add("Documenter")'
-  - julia -e 'cd(Pkg.dir("StaticArrays")); include(joinpath("docs", "make.jl"))'
+  - julia -e 'using Pkg; Pkg.add("Documenter")'
+  - julia -e 'import StaticArrays; cd(joinpath(dirname(pathof(StaticArrays)), "..")); include(joinpath("docs", "make.jl"))'
```
Comment (fredrikekre) [inline .travis.yml]: This line can just be 
```
- julia docs/make.jl
```

**Comment by mschauer** [comment]
I still build the docs under 0.7, but added 1.0 to the build matrix and the docs parse successfully there too.

**Pull request merged**